uaa-server/uaa.yml

```diff
@@ -0,0 +1,204 @@
+      scim.zones: Control a user's ability to manage a zone
+      scim.invite: Send invitations to users
+      password.write: Change your password
+      oauth.approval: Manage approved scopes
+      oauth.login: Authenticate users outside of the UAA
+      openid: Access profile information, i.e. email, first and last name, and phone number
+      groups.update: Update group information and memberships
+      uaa.user: Act as a user in the UAA
+      uaa.resource: Serve resources protected by the UAA
+      uaa.admin: Act as an administrator throughout the UAA
+      uaa.none: Forbid acting as a user
+      uaa.offline_token: Allow offline access
+      order.me: Permission to read personal orders
+      order.admin: Permission to read all orders
+      menu.read: Permission to read all menu items
+      menu.write: Permission to create, update, and delete menu items
+    users:
+      - admin|password|admin@testapp.com|Administrative|Account|uaa
+      - customer|password|customer@testapp.com|John|Doe|menu.read,order.me|uaa
+      - manager|password|manager@testapp.com|Jonathan|Doe|menu.read,menu.write,order.admin|uaa
+    userids_enabled: true
+    user:
+      override: true
+
+  require_https: false
+
+  oauth:
+    user:
+      authorities:
+        - openid
+        - scim.me
+        - cloud_controller.read
+        - cloud_controller.write
+        - cloud_controller_service_permissions.read
+        - password.write
+        - scim.userids
+        - uaa.user
+        - approvals.me
+        - oauth.approvals
+        - profile
+        - roles
+        - user_attributes
+        - uaa.offline_token
+    clients:
+      admin-portal:
+        authorized-grant-types: authorization_code
+        scope: openid,menu.read,menu.write,order.admin
+        authorities: uaa.resource
+        redirect-uri: http://localhost:63757/signin-cloudfoundry
+        app-launch-url: http://localhost:63757/Home/
+        show-on-homepage: true
+        secret: adminportal_secret
+        description: "UI Admin Portal for administering orders"
+      order-service:
+        authorized-grant-types: client_credentials
+        scope: openid,menu.write,order.admin
+        authorities: uaa.resource
+        secret: orderservice_secret
+        description: "API Service for administering orders"
+      customer-portal:
+        secret: secret
+        authorized-grant-types: authorization_code
+        scope: openid,menu.read,order.me
+        authorities: uaa.resource
+        redirect-uri: http://localhost:8082/login
+
+  jwt:
+    token:
+      refresh:
+        format: opaque
+      policy:
+        accessTokenValiditySeconds: 43200
+        refreshTokenValiditySeconds: 2592000
+        global:
+          accessTokenValiditySeconds: 43200
+          refreshTokenValiditySeconds: 2592000
+        activeKeyId: uaa-jwt-key-1
+        keys:
+          uaa-jwt-key-1:
+            signingKey: |
+              -----BEGIN RSA PRIVATE KEY-----
```

Comment:
Is there security risk with exposing private key?

Reply:
This is just for a sample (not to be used in a real environment) so it should be okay. Otherwise we would probably need to get this generated with UAA cli before use.